src/test/java/org/kohsuke/github/AppTest.java

```diff
@@ -861,10 +862,27 @@ public void notifications() throws Exception {
         for (GHThread t : gitHub.listNotifications().nonBlocking(true).read(true)) {
             if (!found) {
                 found = true;
+                // both thread an unread are included
```

Comment:
I think this is a typo. It shows up again on 879. Probably meant to be this?
```suggestion
                // both read and unread are included
```